install.sh

```diff
@@ -1101,7 +1107,7 @@ spec:
       hostPID: true
       containers:
         - name: chaos-daemon
-          image: pingcap/chaos-daemon:${VERSION_TAG}
+          image: ${DOCKER_REGISTRY_PREFIX}/pingcap/chaos-daemon:${VERSION_TAG}
```

Comment:
when `DOCKER_REGISTRY_PREFIX` is empty, `/pingcap/chaos-daemon:latest` will be an invalid image name.****

Reply:
Ok , i will optimize it.

Reply:
I have set default value `docker.io`